README.md

```diff
@@ -31,7 +31,7 @@ Clone the Git repository and run the following commands:
 npm install
 export DISCORD_TOKEN="<your Discord bot token>"
 export GOOGLE_TOKEN="<your Google (Maps) API key>"
-export CR_ADDRESS="http://media.wizards.com/2018/downloads/MagicCompRules%2020180608.txt"
+export CR_ADDRESS="https://media.wizards.com/2020/downloads/MagicCompRules%2020200417.txt"
```

Comment:
You should also update [line 9 in cr.js](https://github.com/bra1n/judgebot/pull/129/files#diff-d4f5e35b892d3603ca531ef9da4b2964L9)